app/models/subscriber_list_params/single_page_lists.rb

```diff
@@ -16,7 +18,33 @@ def list_params
         "title" => content_item["title"],
         "content_id" => content_item["content_id"],
         "description" => content_item["description"],
-      }
+      }.merge(strip_if_empty_value(links))
+    end
+
+  private
+
+    def links
+      { "links" => strip_if_empty_key(reverse_links) }
+    end
+
+    def reverse_links
+      { reverse_linked_key => [content_item["content_id"]] }
+    end
+
+    def reverse_linked_key
+      DOCUMENT_TYPES_PERMITTED_TO_HAVE_REVERSE_LINKED_LISTS[document_type] || ""
```

Comment:
```suggestion
      DOCUMENT_TYPES_PERMITTED_TO_HAVE_REVERSE_LINKED_LISTS[document_type]
```

I imagine it doesn't matter if this returns nil?

Reply:
Ah yes you're right, thank you